Exiled.Events/Handlers/Player.cs

```diff
@@ -8,6 +8,7 @@
 namespace Exiled.Events.Handlers
 {
     using System;
+    using System.Collections.Generic;
```

Comment:
why

Reply:
Not sure what you mean, you need using

Reply:
Yeah, it's not needed here.

Reply:
![image](https://user-images.githubusercontent.com/24619207/177051251-f1f53bd8-3e9b-47d8-b343-9419c62a2eb9.png)
Hmm are you sure?